invenio_vocabularies/contrib/funders/mappings/os-v2/funders/funder-v2.0.0.json

```diff
@@ -40,11 +40,11 @@
         }
       },
       "filter": {
-        "lowercase": {
+        "rdm_lowercase": {
           "type": "lowercase",
           "preserve_original": true
         },
-        "asciifolding": {
+        "rdm_asciifolding": {
```

Comment:
Hello! Sorry for the nitpick: We are trying to keep the settings same across all mappings so can we keep the name [asciifoldingpreserveoriginal](https://github.com/inveniosoftware/invenio-vocabularies/blob/9f69aa7a46da9c1168712157a0189878288f7c0a/invenio_vocabularies/contrib/names/mappings/os-v2/names/name-v2.0.0.json#L47)? Same for lowercasepreserveoriginal.

 However, the change looks good to go!

Reply:
Hi, yes sure, I wasn't aware there was a solution elsewhere. I'll change the filter names.